src/game/game.service.ts

```diff
@@ -45,8 +54,12 @@ export class GameService {
     );
     if (filteredPlayers.length || game.status === 'ACTIVE')
```

Comment:
It`s better to use 'some' method of Array, if you are only checking for user presence